topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: tutorial_hands_on
+
+title: Data submission using ENA upload Tool
+questions:
+- How to prepare sequences for submission to ENA?
+- How to upload raw sequences to ENA?
+objectives:
+- Manage sequencing files (AB1, FASTQ, FASTA, FASTQ.GZ)
+- Clean sequences in an automated and reproducible manner
+- Perform alignments for each sequence
+- Have the necessary sequence format to submit to ENA
+- Submit raw reads to ENA using the ENA upload Tool
+time_estimation: 2h
+key_points:
+- Quality Control on Sequences

```

Comment:
key points should be statements, full sentences, think of them as the lessons learned from your tutorial.

Reply:
Hope this is better like this ;)